client/src/components/SectionTable/SectionTable.js

```diff
@@ -82,42 +84,19 @@ class SectionTable extends PureComponent {
                     {/* <AlmanacGraph courseDetails={courseDetails} />  */}
 
                     {courseDetails.prerequisiteLink ? (
-                        <Typography variant="h6" style={{ flexGrow: '2', marginTop: 9 }}>
-                            <a
-                                target="blank"
-                                style={{
-                                    textDecoration: 'none',
-                                    color: '#72a9ed',
-                                }}
-                                href={courseDetails.prerequisiteLink}
-                                rel="noopener noreferrer"
-                            >
-                                Prerequisites
-                            </a>
-                        </Typography>
+                        <CourseInfoButton
+                            text="Prerequisites"
+                            icon={<PriorityHigh />}
```

Comment:
Hmm actually I'm not a huge fan of the `!` 

Maybe we should use this instead? https://mui.com/components/material-icons/?selected=TaskAlt
![image](https://user-images.githubusercontent.com/29494270/153740946-3fb0538f-990e-4621-9c6b-59ee4a5a10c4.png)


Or here's a few other suggestions:
- https://mui.com/components/material-icons/?selected=GppGood
- https://mui.com/components/material-icons/?selected=FileDownloadDone
- https://mui.com/components/material-icons/?selected=Merge

Reply:
Those icons aren't in the @material-ui/icons library. Assignment might look good too.

https://mui.com/components/material-icons/?selected=Assignment

![image](https://user-images.githubusercontent.com/8692395/153742297-d3d603b9-93c7-4389-8ca0-506d35d11670.png)

Reply:
Ah bummer but yeah I think Assignment does look good! Let's go with that.